packages/react/src/Suspense.en.mdx

```diff
@@ -7,6 +7,14 @@ title: <Suspense/>
 
 @suspensive/react's `<Suspense/>` will be just Suspense of original React.
 
+### `<Suspense.CSROnly/>`
+
+`<Suspense.CSROnly/>` will return fallback first. After mount, return children only in client. but in server, return fallback only.
```

Comment:
How about this?!

```suggestion
`<Suspense.CSROnly/>` will return fallback in server. After mount(in client) return children. 
```

Reply:
Graceful your review. It's so much better